examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -526,39 +526,47 @@
    "source": [
```

Comment:
<p>Could you link to the other examples with the advanced use cases?</p><p> </p><p>Nit:"You may" is used at the beginning of both sentences. Maybe "You also have the option to customize the default locations for checkpoints and output data." would read better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>